test/files/preload.js

```diff
@@ -0,0 +1,21 @@
+module.exports = exports = function() {
+    window.preloadNumber = 7; 
+
+    window.prompt = function(message, defaultResponse) {
+        var response = defaultResponse;
+        if(message == 'foo'){
+            response =  'bar';
+        }
+        __nightmare.ipc.send('page', 'prompt', message, response);
+        return response;
+    };
+
+    window.confirm = function(message) {
+        var response = false;
+        if(message == 'foo'){
+            response = true;
+        }
+        __nightmare.ipc.send('page', 'confirm', message, response);
```

Comment:
we should update this test to reflect the new `require('{{!userPreloadFilepath}}')(__nightmare.ipc);`